data/brands/shop/greengrocer.json

```diff
@@ -191,6 +191,27 @@
         "name:zh": "钱大妈",
         "shop": "greengrocer"
       }
+    },
+    {
+      "displayName": "鲜丰水果",
+      "id": "xianfengfruit-c3a176",
+      "locationSet": {
+        "include": ["cn"],
+        "exclude": ["hk", "mo"]
+      },
+      "tags": {
+        "brand": "鲜丰水果",
+        "brand:en": "Xianfeng Fruit",
+        "brand:wikidata": "Q136192256",
+        "brand:zh-Hans": "鲜丰水果",
+        "brand:zh-Hant": "鮮豐水果",
+        "delivery": "yes",
+        "name": "鲜丰水果",
+        "name:en": "Xianfeng Fruit",
+        "name:zh-Hans": "鲜丰水果",
```

Comment:
I'd suggest use `name:zh` not those 2 separated tags because Mainland of China mainly use Simplified script, and haven't got this tag may led user's display language fallback to English.